doc/Language/faq.pod6

```diff
@@ -299,6 +299,77 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+
+=head1 Common operations
+
+=head2 String: How can I parse and get a L<number|/language/numerics> from a L<string|/language/type/Str>
+
+Use the L<+ prefix|/language/operators#prefix_+>:
+
+=begin code :ok-test<dd>
+say "42.123456789123456789";  # OUTPUT: «42.123456789123456789␤»
```

Comment:
why the ok-test<dd>?

Reply:
I thought it was used for automatic testing. What is its use ?

Reply:
It's in the [EXAMPLES guide](https://github.com/Raku/doc/blob/master/writing-docs/EXAMPLES.md)
There're no automatic tests of examples, it would take too long, but you can use author tests to test the new examples.